### ISSUE TITLE
Desktop configuration for multiple monitors wrong

### ISSUE THREAD

**Issue by njdart** [baskerville/bspwm#654]
I'm trying to configure bspwm for multiple monitors, and I can't seem to get it to respect which desktops are on which monitor. [This image](http://i.imgur.com/ElRL44D.png) shows the layout i'm trying to achieve, with desktops 1-3 on the left, 4-7 in the center, and 8-10 on the right. The relevant parts of my ```bspwmrc``` and ```sxhkdrc``` are below. This output gives me desktops that are positioned correctly (DVI-D-0 is indeed to my left, etc...) but bspwm configures the desktops backwards. Desktops ```I II III``` are on my right and ```VIII IX X``` are on my left. Using ```bspc monitor right -f``` to focus the right monitor,  it does select the monitor on the right, but ```bspc desktop -f '^1'``` selects a desktop on my right monitor, not the left as I expect

[bspwmrc](https://github.com/njdart/dot-files/blob/master/bspwm/bspwmrc):
```sh
xrandr --output DVI-D-0 --mode 1920x1080 --pos 0x0      --rotate left \
       --output HDMI-0  --mode 1920x1080 --pos 1080x300 --rotate normal --primary \
       --output DVI-I-1 --mode 1920x1080 --pos 3000x0   --rotate right

# Rename monitors
bspc monitor DVI-D-0 -n left
bspc monitor HDMI-0 -n center
bspc monitor DVI-I-1 -n right

bspc monitor left -d I II III
bspc monitor center -d IV V VI VII
bspc monitor right -d VIII IX X
```

[sxhkdrc](https://github.com/njdart/dot-files/blob/master/bspwm/sxhkdrc#L63):
```sh
# Move to, push node to, desktop
super + {_,shift + }{1-9,0}
  bspc {desktop -f,node -d} ^{1-9,10}
```

Using version ```0.9.2-49-gfff172e```

**Comment by baskerville** [comment]
What's the output of `bspc wm -d | jq '.monitors[] | {name, rectangle}'`?

**Comment by njdart** [comment]
It appears to describe what I expect;

```json
{
  "name": "right",
  "rectangle": {
    "x": 3000,
    "y": 0,
    "width": 1080,
    "height": 1920
  }
}
{
  "name": "center",
  "rectangle": {
    "x": 1080,
    "y": 300,
    "width": 1920,
    "height": 1080
  }
}
{
  "name": "left",
  "rectangle": {
    "x": 0,
    "y": 0,
    "width": 1080,
    "height": 1920
  }
}
```

[Full Output of ```bspc wm -d```](https://gist.github.com/njdart/ef3b41911db50e2ddd70d74e7c3c55f8)

**Comment by njdart** [comment]
I should clarify, the output of ```bspc wm -d``` appears correct for the configuration I desire, but I am still focusing on the wrong desktop when invoking ```bspc desktop -f '^1'``` (should be the first desktop on my left monitor, but is in fact the first desktop on my right monitor). Is there any way to add further debug to bspwm/sxhkd?

**Comment by baskerville** [comment]
This issue is fixed by 4b035e0. Have you recompiled?

After having relaunched *bspwm*, the output of `bspc query -M --names` should now be:
```
left
center
right
```

**Comment by njdart** [comment]
My apologies, I didn't notice the commit that closed this issue.

It is indeed fixed now, thanks :+1:
